modules/op-disabling-pipeline-templates.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-disabling-pipeline-templates_{context}"]
+= Disabling pipeline templates
+
+By default, the `TektonAddon` custom resource (CR) installs pipeline templates with {pipelines-shortname} on the cluster.
```

Comment:
We have this in the docs:
```
You can set the value of the pipelinesTemplates parameter to true only when the value of the clusterTasks parameter is true.
```
Does that mean that we should remove mentions of pipeline templates as well? Or will it work without cluster tasks?